### ISSUE TITLE
No place to add/edit mining pools

### ISSUE THREAD

**Issue by farshadseifi** [FxPool/FxMinerProxyV3#6]
It would be better to add some place in UI to add or edit mining pools from dashboard.
till version 8.0.3 it could be edit from "/etc/fxpool-fxminerproxyv3/localcoinconfig.json" but now in version 8.1.2 it can't be edit or add from there either.
the only solution that I found is create a coin port from dashboard and then edit the "localcoinconfig.json" file and reboot to take efect.

**Comment by FxPool** [comment]
Yes, because we are going to add the function of editing the target mining pool in the UI, but the UI interface has not been developed yet. We will update this function at the end of September. In fact, the target mining pool can be entered (do you only use the drop-down menu function?)

**Comment by farshadseifi** [comment]
thank you so much for fast development. I haven't noticed that we can write the pool address. I just used only drop-down menu and edit it from "localcoinconfig.json" :D. now I checked and know that pool address can be entered from text box either.
